lib/classes/Variables.js

```diff
@@ -145,7 +148,7 @@ class Variables {
   }
 
   getValueFromSelf(variableString) {
-    let valueToPopulate = _.cloneDeep(this.service);
+    let valueToPopulate = this.service;
```

Comment:
@eahefnawy for me it makes sense to get rid of deep cloning to prevent an infinite loop / recursion.

What's your take on this since you've implemented it in the first place?